Automattic/jetpack
# Boost: Refine badges for features

Fixes https://github.com/Automattic/jetpack/issues/40025

## Proposed changes:

* Add "Upgraded" badge to cornerstone pages.
* Change "Upgraded" badge to share the same component as the "Experimental" one.

![CleanShot 2024-11-04 at 12 55 04](https://github.com/user-attachments/assets/eff26c27-85a7-4432-8ead-d3badd8a2dfd)

### Other information:


## Jetpack product discussion

n/a

## Does this pull request change what data or activity we track or use?

no

## Testing instructions:

* Setup Boost (premium if you want to see the badge next to Cornerstone Pages);
* Make sure the badges in the UI look okay.
